Code/Tools/ProjectManager/Platform/Linux/ProjectUtils_linux.cpp

```diff
@@ -26,32 +26,76 @@ namespace O3DE::ProjectManager
+            auto queryCmakeVersionQuery = ExecuteCommandResult(cmakeInstalledPath, QStringList{ "--version" });
+            if (queryCmakeVersionQuery.IsSuccess())
+            {
+                AZ_TracePrintf(
+                    "Project Manager", "\"%s\" detected.", queryCmakeVersionQuery.GetValue().split("\n")[0].toUtf8().constData());
+            }
+
+            return AZ::Success(QString{ cmakeInstalledPath });
+        }
+
+        AZ::Outcome<QString, QString> FindSupportedCompilerForPlatform()
+        {
+            // Query the version of cmake that is installed
+            if (auto queryCmakeVersionQuery = FindSupportedCMake(); !queryCmakeVersionQuery.IsSuccess())
+            {
+                return queryCmakeVersionQuery;
             }
 
-            // Look for the first compatible version of clang. The list below will contain the known clang compilers that have been tested for O3DE.
+            // Look for the first compatible version of clang. The list below will contain the known clang compilers that have been tested
+            // for O3DE.
             for (const QString& supportClangVersion : SupportedClangVersions)
             {
-                auto whichClangResult = ProjectUtils::ExecuteCommandResult("which", QStringList{QString("clang-%1").arg(supportClangVersion)});
-                auto whichClangPPResult = ProjectUtils::ExecuteCommandResult("which", QStringList{QString("clang++-%1").arg(supportClangVersion)});
+                auto whichClangResult =
+                    ProjectUtils::ExecuteCommandResult("which", QStringList{ QString("clang-%1").arg(supportClangVersion) });
+                auto whichClangPPResult =
+                    ProjectUtils::ExecuteCommandResult("which", QStringList{ QString("clang++-%1").arg(supportClangVersion) });
                 if (whichClangResult.IsSuccess() && whichClangPPResult.IsSuccess())
                 {
                     return AZ::Success(QString("clang-%1").arg(supportClangVersion));
                 }
             }
-            return AZ::Failure(QObject::tr("Clang not found. <br><br>"
-                "Make sure that the clang is installed and available from the command prompt. "
-                "Refer to the <a href='https://o3de.org/docs/welcome-guide/setup/requirements/#cmake'>O3DE requirements</a> page for more information."));
-        }
 
+            // Next detect clang executables without a compiler version as a backup
+            // Only Ubuntu has clang++-<version> symlinks, other linux distros do not
+            auto whichClangNoVersionResult = ProjectUtils::ExecuteCommandResult("which", QStringList{ QString("clang") });
+            auto whichClangPPNoVersionResult = ProjectUtils::ExecuteCommandResult("which", QStringList{ QString("clang++") });
+            if (whichClangNoVersionResult.IsSuccess() && whichClangPPNoVersionResult.IsSuccess())
+            {
+                return AZ::Success(QString("clang"));
+            }
```

Comment:
We could consolidate to just 1 loop if we replace (on top)
```
const QStringList SupportedClangVersions = {"13", "12", "11", "10", "9", "8", "7", "6.0"};
```
with
```
const QStringList SupportedClangVersions = {"-13", "-12", "-11", "-10", "-9", "-8", "-7", "-6.0", ""};
```

and 
```
QString("clang-%1").arg(supportClangVersion)
...
QString("clang++-%1").arg(supportClangVersion)
```
with
```
QString("clang%1").arg(supportClangVersion)
...
QString("clang++%1").arg(supportClangVersion)
```

Reply:
I'll make that change.